website/content/docs/upgrading/index.mdx

```diff
@@ -25,6 +25,10 @@ supported. The upgrade notes for each intervening version must be reviewed. The
 upgrade notes may describe additional steps or configuration to update before,
 during, or after the upgrade.
 
+You should also consult the [deprecation notices](/vault/docs/deprecation), which
+contains a comprehensive list of all deprecated features, and which versions they
+are removed (or scheduled to be removed).
```

Comment:
```suggestion
--------------------------------------------------------------------------------
We also recommend you consult the
[deprecation notices](/vault/docs/deprecation). The notice page includes
a comprehensive list of deprecated features and the Vault versions where
the feature was removed or is scheduled to be removed.
```
Style correction: write in active voice, frame recommendations as recommendations, not commands